XCUITest/ShortcutsTest.swift

```diff
@@ -12,54 +12,67 @@ class ShortcutsTest: BaseTestCase {
         super.tearDown()
     }
 
-    func testAddRemoveShortCut() {
+    func testAddRemoveShortcut() {
         addShortcut(website: "mozilla.org")
 
         // Tap on erase button to go to homepage and check the shortcut created
         app.eraseButton.tap()
 
         // Verify the shortcut is created
-        waitForExistence(app.otherElements.staticTexts["M"], timeout: 5)
         XCTAssertTrue(app.otherElements.staticTexts["Mozilla"].exists)
 
         // Remove created shortcut
-        app.otherElements.staticTexts["M"].press(forDuration: 2)
+        app.otherElements["outerView"].press(forDuration: 2)
         waitForExistence(app.collectionViews.cells.buttons["Remove from Shortcuts"])
         app.collectionViews.cells.buttons["Remove from Shortcuts"].tap()
-        waitForNoExistence(app.otherElements.staticTexts["M"])
+        waitForNoExistence(app.otherElements.staticTexts["Mozilla"])
         XCTAssertFalse(app.otherElements.staticTexts["Mozilla"].exists)
     }
 
+    func testAddRenameShortcut() {
+        addShortcut(website: "mozilla.org")
+
+        // Tap on erase button to go to homepage and check the shortcut created
+        app.eraseButton.tap()
+
+        // Verify the shortcut is created
+        XCTAssertTrue(app.otherElements.staticTexts["Mozilla"].exists)
+
+        // Rename shortcut
+        app.otherElements["outerView"].press(forDuration: 2)
+        waitForExistence(app.collectionViews.cells.buttons["Rename Shortcut"])
+        app.collectionViews.cells.buttons["Rename Shortcut"].tap()
+        let textField = app.alerts.textFields.element
+        textField.clearAndEnterText(text: "Cheese")
+        app.alerts.buttons["Save"].tap()
+
+        waitForExistence(app.otherElements.staticTexts["Cheese"])
+        XCTAssertTrue(app.otherElements.staticTexts["Cheese"].exists)
+    }
+
     func testAdd4Shortcuts() {
         addShortcut(website: "mozilla.org")
-//        addShortcut(website: "example.com")
-//        addShortcut(website: "pocket.com")
-//        addShortcut(website: "wikipedia.com")
+        addShortcut(website: "example.com")
```

Comment:
You will need to comment these three lines...for some reason we can add one shortcut via de automated tests but I can't go back to an state from I can add another one... that's why these were commented out.